### ISSUE TITLE
Notification - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#5013]
Add Accessibility information to the Notification page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/5004/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [ ] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

When your work is ready for review please get a first pass review from another member of the accessibility sprint and once that is done reach out to Jessica to flag that a final review is needed.

Resources and videos for this work can be found here: https://hpe-my.sharepoint.com/shared?id=%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents%2FHPE%20Design%20System%2FAccessibility&listurl=https%3A%2F%2Fhpe%2Esharepoint%2Ecom%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents

**Comment by SeamusLeonardHPE** [comment]
https://v2.grommet.io/notification?theme=hpe

https://storybook.grommet.io/?path=/story/visualizations-notification-global--global&globals=theme:hpe

https://design-system.hpe.design/components/notification

**Comment by luketa8** [comment]
First pass review done. Thanks!

**Comment by SeamusLeonardHPE** [comment]
Thanks @luketa8 

@jcfilben @taysea can you have a look at notification for a final pass on the scoring (I may ask for guidance on any notes that we should publish after your review)

**Comment by britt6612** [comment]
@SeamusLeonardHPE 
left comments on your figma to add issues

**Comment by britt6612** [comment]
@SeamusLeonardHPE 

Just looks like we need to file these last 3 issues
1.4.6 
2.4.9
2.5.5

**Comment by britt6612** [comment]
@SeamusLeonardHPE thank you! closing issue now
